projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/dynamic-multi-select-list/write-dynamic-multi-select-list-field.component.spec.ts

```diff
@@ -146,7 +155,7 @@ describe('WriteDynamicMultiSelectListFieldComponent', () => {
       });
     });
 
-    it('should show a link in the checkbox label', () => {
+    xit('should show a link in the checkbox label', () => {
```

Comment:
This functionality is required to work. If it's not please raise a ticket to get it fixed

Reply:
As discussed, the test needs to reviewed.  The manual application of the scenario in the app works fine - https://tools.hmcts.net/jira/browse/EXUI-1903